articles/iot-hub/iot-hub-tls-support.md

```diff
@@ -126,9 +126,13 @@ IoT Edge devices can be configured to use TLS 1.2 when communicating with IoT Hu
 
 After a successful TLS handshake, IoT Hub can authenticate a device using a symmetric key or an X.509 certificate. For certificate-based authentication, this can be any X.509 certificate, including ECC. IoT Hub validates the certificate against the thumbprint or certificate authority (CA) you provide. To learn more, see [Supported X.509 certificates](iot-hub-dev-guide-sas.md#supported-x509-certificates).
 
-## Mutual TLS support
+## Mutual TLS authentication support
```

Comment:
Since you changed the header, you will need to also change the bookmark link in iot-edge-certs.md that the build engine flagged as a warning.

Reply:
Hi Kelly, I cant find a reference to this link in the IoT Edge docs anymore. Could it have been removed or maybe I'm missing something?